src/Session.js

```diff
@@ -38,14 +38,24 @@ const Session = class Session {
         });
     }
 
-    markAccessed(modelName) {
-        this.getDataForModel(modelName).accessed = true;
+    markAccessed(modelName, modelIds) {
```

Comment:
food fort thought:

1) Make modelIds optional. If not provided, grab all the IDs from the model and use those.
2) This would make this change backwards compatible in case people called markAccessed from within their apps

Reply:
Well they also get assigned to this (see [here](https://github.com/B-Reif/redux-orm/blob/75f2323261ddf0b7e05ea8b991906e40f1eac564/src/Session.js#L32)), so you could do:

```js
   idsFromModelName(modelName) {
     return this[modelName].all().toRefArray().map(r => r.id);
    }
```

i think...

Reply:
Actually I don't think this would even work in this context since `toRefArray` starts a query itself, which would in turn run `markAccessed` itself.

Reply:
oh, interesting, didn't know that about toRefArray...